src/Chartjs/Resources/assets/dist/controller.js

```diff
@@ -1,16 +1,12 @@
 import { Controller } from '@hotwired/stimulus';
 import { Chart } from 'chart.js';
 
-class controller extends Controller {
+class default_1 extends Controller {
     connect() {
         if (!(this.element instanceof HTMLCanvasElement)) {
             throw new Error('Invalid element');
         }
-        const viewData = this.element.getAttribute('data-view');
-        if (!viewData) {
-            throw new Error('Missing data-view attribute.');
-        }
-        const payload = JSON.parse(viewData);
+        const payload = this.viewValue;
```

Comment:
Built file has a few extra changes because I think we didn't rebuild after the chartjs -> values PR :)